base_exception/models/base_exception.py

```diff
@@ -53,10 +53,6 @@ def check_exception_type_consistency(self):
             if (
                 (rule.exception_type == "by_py_code" and not rule.code)
                 or (rule.exception_type == "by_domain" and not rule.domain)
-                or (
-                    rule.exception_type == "by_method"
-                    and not self._check_method_valid(rule.method, rule.model)
-                )
```

Comment:
Please add back a check on rule.method which can be undefined.
```py
 or (rule.exception_type == "by_method" and not rule.method)
```

Reply:
Thanks for the feedback ! DONE